public/pages/Reindex/container/Reindex/Reindex.tsx

```diff
@@ -550,36 +555,52 @@ class Reindex extends Component<ReindexProps, ReindexState> {
       </CustomFormRow>
     );
 
-    // expand data streams and aliases
-    const allSelectedIndices = this.getAllSelectedIndices();
-
-    return (
-      <div style={{ padding: "0px 50px" }}>
-        <EuiTitle size="l">
-          <h1>Reindex</h1>
-        </EuiTitle>
-        {subTitleText}
-        <EuiSpacer />
-
-        <ContentPanel title="Configure source index" titleSize="s">
-          <EuiSpacer />
+    const description = [
+      {
+        renderComponent: (
           <CustomFormRow
-            label="Specify source indexes or data streams"
-            isInvalid={sourceErr.length > 0}
-            error={sourceErr}
             fullWidth
-            helpText="Specify one or more indexes or data streams you want to reindex from."
+            label=""
+            helpText={
+              <div>
+                Use reindex to make extensive changes to your index. Reindex will copy data of the source index into another index.{" "}
+                <EuiLink href={this.context.docLinks.links.opensearch.reindexData.base} target="_blank" rel="noopener noreferrer">
+                  Learn more
+                </EuiLink>
+              </div>
+            }
```

Comment:
Is this a description? If so, please use `TopNavControlDescriptionData`. The one used, `TopNavControlComponentData`, is for extremely rare situations where existing controls don't provide what you need. All other types of `TopNavControlData` get globally styles but `TopNavControlComponentData` by their nature cannot.

Reply:
Hey Miki, we actually could not figure out a way to include the link in our description using `TopNavControlDescriptionData` and hence have used `TopNavControlComponentData` instead. If there is a way to include the link using it, can you please help us on how to do it?

Reply:
Hi Miki, we actually tried that but the EUI link which we were getting looked a bit too spaced out hence did this. PFA the screenshots:
With current code:
<img width="1682" alt="image" src="https://github.com/user-attachments/assets/8e1af670-3641-41d0-b127-2cae53a7245f">
With the suggested code:
<img width="1677" alt="image" src="https://github.com/user-attachments/assets/b21bf58c-9210-4bdb-8d75-cff18f1326bf">
Please let us know if we are fine with the UI or know a fix to remove the spaces. In that case we need to update all the description in ISM that we can take up as follow up PR

Reply:
I have edited it as per your suggestion but would be great if you could clarify it once. Thanks!